pkg/services/alerting/notifiers/pagerduty.go

```diff
@@ -99,6 +95,26 @@ func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
 	pn.log.Info("Notifying Pagerduty", "event_type", eventType)
 
 	payloadJSON := simplejson.New()
+	bodyJSON := simplejson.New()
+
+	for _, tag := range evalContext.Rule.AlertRuleTags {
+		customData.Set(tag.Key, tag.Value)
+
+		// Override tags appropritatly if the are in the Pagerduty v2 API
+		if strings.ToLower(tag.Key) == "group" {
+			payloadJSON.Set("group", tag.Value)
+		}
+
+		if strings.ToLower(tag.Key) == "class" {
```

Comment:
Isn't it better to use the `else if` form in order to improve readability?

Reply:
```
switch strings.ToLower(tag.Key) {
  case "group":
    ....
```

Reply:
yea, I wasnt familiar w/ the `fallthrough` operator in a go switch.... learn something new every day :)